grep-0001-coding-guidelines.md

```diff
@@ -73,6 +72,20 @@ and follow them.
   can thus be optimized better at compile time. In many cases, lambdas are
   also more readable than the `bind` calls, which have add their own
   syntactical idiosyncrasies (such as the `_1` argument etc.).
+* Feel free to use modern C/C++ features even if they were not used before.
+  Make sure they work with the compilers and dependencies which are set for the
+  version of GNU Radio the commit will be made upon. The GNU Radio CI system
+  will be able to confirm this. (Note: C++11 features are available starting
+  with GNU Radio version 3.8).
+
+#### Code Style Guidelines
```

Comment:
Can we call formatting rules formatting rules. "Coding Style" is ambiguous, for example, I prefer a functional coding style, which is not what this means.

Reply:
:+1: Will do that.